src/docs/cmdstan-guide/getting-started.tex

```diff
@@ -186,10 +186,11 @@ \subsection{Compiling a Stan Program}
 to an executable.)
 
 A Stan program must be in a file with the file extension
-\code{.stan}. To create an executable from a Stan program,
-\code{make} will be called with the name of the executable as its
-argument. For Mac and Linux, it is the name of the Stan program with
-the \code{.stan} omitted. For Windows, replace \code{.stan} with
+\code{.stan}. To create an executable from a Stan program, \code{make}
+will be called with the name of the executable as its argument. For
+Mac and Linux, it is the name of the Stan program with the
+\code{.stan} omitted. For Windows, replace .stan with .exe, and make
+sure that the path is given with slashes and not backslashes.
 \code{.exe}.
```

Comment:
Looks like this line shouldn't be in there any more?  And the .stan and .exe should have \code{} around them